bpm/bonita-core/bonita-process-engine/src/main/java/org/bonitasoft/engine/api/impl/PlatformAPIImpl.java

```diff
@@ -655,9 +644,7 @@ private void createDefaultTenant(final PlatformServiceAccessor platformAccessor,
             sessionAccessor.deleteSessionId();
             sessionAccessor.setSessionInfo(session.getId(), tenantId);// necessary to create default data source
 
-            // Create default profiles
-            createDefaultProfiles(tenantServiceAccessor);
-
+            // Create default profiles: done by profile updater restart handler
```

Comment:
In Community I did not implement it here since the restart handler will also perform the update at platform startup and this is the only time when the default tenant can be created.

In SP however, the tenant can be created at any time without a restart so the call to this method needs to be performed at tenant creation.